publisher-sdk/proguard-rules.pro

```diff
@@ -42,38 +42,6 @@
 
 # Leaving Criteo SDK models.
 -keepparameternames
```

Comment:
In addition to this rule, I would suggest to add few [`-keepattributes`](https://www.guardsquare.com/en/products/proguard/manual/usage#keepattributes). See https://www.guardsquare.com/en/products/proguard/manual/examples#library for instance.

Reply:
Done